### DISCUSSION TITLE
Unable to run a nextjs project

### DISCUSSION THREAD

**Discussion by elshadayzb** [vercel/next.js#48332] [Help]
### Summary

I could not create and run nextjs project in any folder containing Users in the path.

Created a nextjs project in C:\Users\next_project folder and tried to execute npm run dev but throws the error, TypeError: api.createContextKey is not a function. Then, moved the project to a different folder C:\next_projects and npm run dev is working.
I am working with next v13.3.0.

**Comment by icyJoseph** [comment]
Interesting, that error seems to be related to open telemetry, failing to load, maybe there's some special permissions on that directory?

**Reply by elshadayzb** [reply]
What do you recommend?

**Reply by elshadayzb** [reply]
No, I was able to create and run nextjs project previously. I have a nextjs project with next version 13.0.6 and it is working fine. is this problem associated with the latest next release.

**Reply by icyJoseph** [reply]
Well, as I said, that particular error is coming from not being able to load open telemetry. Do you have that installed somewhere in your system? Do consider that it might have been added recently, which is why jumping from 13.0.6 to 13.3 would cause it.

**Reply by elshadayzb** [reply]
No, I haven't, I am still facing the same problems and moved to using NextJs version 13.2. Versions higher than 13.2 are throwing the same errors.

**Comment by sushanyadav** [comment]
Got similar issue:
![CleanShot 2023-07-21 at 4 32 51](https://github.com/vercel/next.js/assets/53527664/afe5956b-d397-49c4-971f-2195388528a7)

To resolve I had to install all of these additional packages:
https://github.com/vercel/next.js/issues/48118#issuecomment-1538628004

I wonder if there's any other solution.

**Comment by tkejr** [comment]
Same issue
